resolwe_bio/management/commands/generate_samples.py

```diff
@@ -264,9 +271,9 @@ def create_data(self, reads_name='seq_reads', annotated=False, rseed=None):
             sample.descriptor = self.generate_sample_desciptor(d.name)
             sample.presample = False
             sample.save()
-            print "Created sample: {} (id={})".format(sample.name, sample.id)
+            self.stdout.write("Created sample: {} (id={})".format(sample.name, sample.id))
```

Comment:
You could include `from __future__ import absolute_import, division, print_function, unicode_literals` in the first line and then just use print(...) for Py2 and Py3. Even better would be to use `logger.info`. See and [example of logger usage here](https://github.com/genialis/resolwe/blob/a7a6b1eb22d61dfc385735f997dcaa920676e1f4/resolwe/flow/executors/__init__.py). Domen can tell you more about logger and logging.